mesonbuild/dependencies/data/CMakeListsLLVM.txt

```diff
@@ -1,4 +1,3 @@
-cmake_minimum_required(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION} )
```

Comment:
You're going to need to manually patch this in the old version of meson, since one of the changes that broke python 3.6 support was beginning to depend on importlib.resources to use bundled data files instead of syncing their contents into an inline string.

There's a script to update this for you.

Reply:
Basically just add an additional commit "run tools/gen_data.py and commit the results".